superset-frontend/spec/javascripts/components/AnchorLink_spec.jsx

```diff
@@ -18,7 +18,7 @@
  */
 import React from 'react';
 import { shallow } from 'enzyme';
-import sinon from 'sinon';
+import { promiseTimeout } from '@superset-ui/core';
```

Comment:
Could use `async` pattern similar to other ones for consistency.

Reply:
fair enough